openedx/features/content_type_gating/partitions.py

```diff
@@ -44,7 +44,10 @@ def create_content_gating_partition(course):
     Create and return the Content Gating user partition.
     """
 
-    if not ContentTypeGatingConfig.enabled_for_course(course_key=course.id):
+    enabled_for_course = ContentTypeGatingConfig.enabled_for_course(course_key=course.id)
+    studio_override_for_course = ContentTypeGatingConfig.current(course_key=course.id).studio_override_enabled
+    LOG.error('%s %s %s', course.id, enabled_for_course, studio_override_for_course)
```

Comment:
are we always logging an error?

Reply:
That was left-over debugging, and is now removed.